### PR TITLE
docs:  Add dynamic plugin setup/installation instruction for the janus-idp plugins

### PR DESCRIPTION
**Pull request by jesuino**
Added brief information about dynamic plugins installation to the README.md, but linking the main Dynamic Plugins docs to avoid information duplication. The README.md instruction is just to let users know about this possibility and direct them to a more complete document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,50 @@ A subset of available Janus IDP plugins is available at our [community site](htt
 
 You can also see the [Plugin Marketplace](https://backstage.io/plugins) for other open-source plugins you can add to your Backstage instance.
 
+## Dynamic Plugins Installation
+
+It is possible to install plugins without code changes in a backstage that supports [Dynamic Plugins](https://github.com/janus-idp/backstage-showcase/blob/main/showcase-docs/dynamic-plugins.md) (e.g. [Janus IDP](https://janus-idp.io/) and [Red Hat Developer Hub](https://developers.redhat.com/rhdh)). Follow the steps below to install a dynamic plugin:
```
Comment (jkilzi) [inline README.md:17]: Are there any prerequisite for enabling the dynamic plugins?
I mean, can a user just use this procedure in a random version of Janus-IDP/Backstage, or should they first make sure a specific version is available, or anything perhaps something else?

Comment (jesuino) [reply README.md:17]: Hello @jkilzi I confirmed that no additional configuration is required to enable dynamic plugins. Regarding what version to use, I know that dynamic plugins are available in RHDH starting on 1.0, but for Janus IDP I recommend users to check if the endpoint `/api/dynamic-plugins-info/loaded-plugins`, if it is available, then you can use dynamic plugins.

Please bear in mind that upstream backstage does not support dynamic plugins ATM.

File: README.md [hunk 2]
```diff
@@ -12,6 +12,50 @@ A subset of available Janus IDP plugins is available at our [community site](htt
...
+
+- Map the dynamic plugins root directory in `app-config.yaml`:
```
Comment (gashcrumb) [inline README.md:19]: `app-config.local.yaml`, sorry missed this one before commenting on the other one :-)

**Comment by jesuino** [comment]
@gashcrumb done, thanks!

**Review by gashcrumb — APPROVED** [review]
:+1:

**Pull request merged**